### PR TITLE
add details to centos_ci proposal

### PR DESCRIPTION
**Pull request by mbukatov**
Adding few details which I found important to directly mention.

### CODE REVIEW

File: specs/centos_ci.adoc [hunk 1]
```diff
@@ -2,28 +2,46 @@
 
 = CentOS CI
 
-CentOS CI should be configured to launch applicable set of tests as an
```
Comment (dahorak) [inline specs/centos_ci.adoc:5]: I think there should be some "Introduction paragraph" (see the sample_template.adoc)

Comment (mbukatov) [reply specs/centos_ci.adoc:5]: ts ts ...

Comment (mbukatov) [reply specs/centos_ci.adoc:5]: I will put something here back again.

Comment (mbukatov) [reply specs/centos_ci.adoc:5]: Fixed.

**Review by dahorak — APPROVED** [review]

**Pull request merged**